justfile

```
@@ -27,6 +27,14 @@ unused:
 doc:
   cargo doc --workspace --open
 
+audit:
+  cargo audit
+
+coverage:
+  # cargo tarpaulin --engine llvm
+  # cargo llvm-cov -- --include-ignored
+  cargo llvm-cov
```

`cargo llvm-cov --html --open` will help developers view coverage details or track changes over time. `coverage-check` prevents accidental coverage regressions. 75% is reasonable based on the ignored integration tests. Audit looks fine.

```suggestion
coverage:
  # cargo tarpaulin --engine llvm
  # cargo llvm-cov -- --include-ignored
  cargo llvm-cov --html --open
  
coverage-check:
  cargo llvm-cov --fail-under-lines 75

coverage-ci:
  cargo llvm-cov --lcov --output-path lcov.info
```